### DISCUSSION TITLE
Do I always have to be logged in as admin?

### DISCUSSION THREAD

**Discussion by CogalTek** [pocketbase/pocketbase#3554] [Q&A] [answered]
Hello, I was wondering if every time I make a request to log in or register I have to be logged in as admin? or can I allow everyone access to my API?

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure I follow. You can control the access to your collections using the API rules - https://pocketbase.io/docs/api-rules-and-filters/ (_by default new collections are "locked", aka. can be accessed only by admins_).

**Reply by CogalTek** [reply]
ok thank's i will read that
